plugins/docker/docker.plugin.zsh

```diff
@@ -0,0 +1,42 @@
+alias di='docker info'
+alias dlg='docker container logs'
+alias dls='docker container ls'
+alias dlsa='docker container ls -a'
+alias dr='docker container run'
+alias drit='docker container run -it'
+alias drm='docker container rm'
+alias drmf='docker container rm -f'
+alias dst='docker container start'
+alias dstp='docker container stop'
+alias dt='docker top'
+alias dv='docker version'
+alias dpo='docker container port'
+alias dpu='docker pull'
+alias dx='docker container exec'
+alias dxit='docker container exec -it'
+alias dbl='docker build'
+alias dhh='docker help'
+alias dcin='docker container inspect'
+alias dpsa='docker container ps -a'
+
+#docker images
+alias dirm='docker image rm'
+alias dils='docker image ls'
+alias dit='docker image tag'
+alias dip='docker image push'
+alias dib='docker image build'
+alias dii='docker image inspect'
+
+
+#docker network
+alias dnls='docker network ls'
+alias dni='docker network inspect'
+alias dnc='docker network create'
+alias dncn='docker network connect'
+alias dndcn='docker network disconnect'
+alias dnrm='docker network rm'
+
+#docker volume
+alias dvls='docker volume ls'
+alias dvi='docker volume inspect'
+alias dvclean='docker volume rm $(docker volume ls -qf dangling=true)'
```

Comment:
There's `docker volume prune` now.

Reply:
Good point. I've changed dvclean to dvprune and used used docker prune instead.